.github/workflows/tpu-tests.yml

```diff
@@ -39,16 +22,16 @@ defaults:
 jobs:
   test-on-tpus:
     runs-on: ubuntu-22.04
-    if: github.event.pull_request.draft == false
+    # run only when merged to master or the PR title contains '[TPU]'
+    if: ${{ github.event.pull_request.merged == true || contains(github.event.pull_request.title, '[TPU]') }}
     env:
       PYTHON_VER: 3.8
     strategy:
-      fail-fast: true
-      max-parallel: 1  # run sequential
+      fail-fast: false
       matrix:
-        # TODO: add also lightning
         pkg-name: ["fabric", "pytorch"]
-    timeout-minutes: 100  # should match the timeout in `tpu_workflow.jsonnet`
+    # 1 hour is the access token lifetime: https://cloud.google.com/docs/authentication/token-types#at-lifetime
+    timeout-minutes: 60  # should match the timeout in the jsonnet files
```

Comment:
I think this is what was causing the "Unauthorized" message. I couldn't find a way to extend the token lifetime